src/frontend/js/components/DashBoard/hooks/useDashboardRoutes.ts

```diff
@@ -0,0 +1,22 @@
+import { useIntl } from 'react-intl';
+import { useMemo } from 'react';
+import slugify from 'slugify';
+import dashboardRoutes, { DashBoardRoute } from 'components/DashBoard/routes';
+
+const useDashBoardRoutes = () => {
+  const intl = useIntl();
+
+  const intlDashboardRoutes = useMemo<Array<DashBoardRoute>>(() => {
```

Comment:
I think `dashboardRoutes` is sufficient. What do you think?
```suggestion
  const dashboardRoutes = useMemo<Array<DashBoardRoute>>(() => {
```